contrib/ci/get-matrix.py

```diff
@@ -26,13 +26,15 @@ def get_fedora_releases(type, exclude=[]):
 
 def get_fedora_matrix():
     fedora_stable = get_fedora_releases('current')
-    fedora_devel = get_fedora_releases('pending', exclude=['eln'])
-    fedora_frozen = get_fedora_releases('frozen', exclude=['eln'])
+    # These are now F41 and f42 that come with sssd-2.10
```

Comment:
I'd prefer to see 'F41' hard coded.
One release later, 'current' will be F41 and it will be broken again.

Reply:
Next release there will be no compatible fedora that i not eol :-) so we can just drop it completely.

Reply:
@pbrezina : I wonder where is the matrix used for builds so we can also switch them off as this seems to affect only the tests.

Reply:
Updated the .github/workflows/copr_build.yml to blacklist explicitly the builds as well.